cli/src/chain_spec/mod.rs

```diff
@@ -188,3 +203,114 @@ pub fn config_genesis(network_keys: NetworkKeys) -> GenesisConfig {
 		}),
 	}
 }
+
+#[cfg(test)]
+pub(crate) mod tests {
```

Comment:
these are new tests introduced?

Reply:
Yes. However more than the tests, the intention for bringing them from Plug was the helper functions in this segment which are used in some other cli tests.